### PR TITLE
Md 4498 drill calcite1.16.0

### PR DESCRIPTION
**Pull request by gparai**
@amansinha100 please review the PR. Thanks!

### CODE REVIEW

File: core/src/main/java/org/apache/calcite/sql/fun/SqlAnyValueAggFunction.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+import org.apache.calcite.sql.type.OperandTypes;
+import org.apache.calcite.sql.type.ReturnTypes;
+
+import com.google.common.base.Preconditions;
+
+/**
+ * Definition of the <code>ANY_VALUE</code> aggregate functions,
+ * returning any one of the values which go into it.
+ *
+ * <p>There are 3 forms:
```
Comment (amansinha100) [inline core/src/main/java/org/apache/calcite/sql/fun/SqlAnyValueAggFunction.java]: Rest of this javadoc comments is unrelated to any_value

File: core/src/main/java/org/apache/calcite/sql/fun/SqlAnyValueAggFunction.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  //~ Constructors -----------------------------------------------------------
+
+  /** Creates a SqlAnyValueAggFunction. */
+  public SqlAnyValueAggFunction(SqlKind kind) {
+    super(kind.name(),
+        null,
+        kind,
+        ReturnTypes.ARG0_NULLABLE_IF_EMPTY,
+        null,
+        OperandTypes.COMPARABLE_ORDERED,
```
Comment (amansinha100) [inline core/src/main/java/org/apache/calcite/sql/fun/SqlAnyValueAggFunction.java]: Is there a reason for making this comparable_ordered ?

File: core/src/main/java/org/apache/calcite/sql/fun/SqlAnyValueAggFunction.java (hunk 1 above)
Comment (gparai) [reply core/src/main/java/org/apache/calcite/sql/fun/SqlAnyValueAggFunction.java]: Artifact of copy paste :( Removed it.

File: core/src/main/java/org/apache/calcite/sql/fun/SqlAnyValueAggFunction.java (hunk 2 above)
Comment (gparai) [reply core/src/main/java/org/apache/calcite/sql/fun/SqlAnyValueAggFunction.java]: No, changed it to ANY.

**Comment by gparai** [comment]
@amansinha100 I have addressed your review comments. Please take a look.

**Comment by amansinha100** [comment]
+1.  Before merging, pls change the commit message to something more meaningful for any_value.
